languages/javascript/reference/keywords/let.md

```diff
@@ -1 +1,32 @@
 # `let`
+
+The `let` keyword declares a block scoped local [variable](https://developer.mozilla.org/en-US/docs/Glossary/Variable), optionally initializing it to a value.
```

Comment:
This should link to _our_ variable reference (abstract, in `/reference`) OR create `variable.md` in `/**/javascript/reference/info`.